apps/emqx/src/emqx_persistent_message_ds_replayer.erl

```diff
@@ -176,9 +177,12 @@ fetch(SessionId, Inflight0, [DSStream | Streams], N, Acc) when N > 0 ->
     #inflight{next_seqno = FirstSeqno, offset_ranges = Ranges} = Inflight0,
     ItBegin = get_last_iterator(DSStream, Ranges),
     {ok, ItEnd, Messages} = emqx_ds:next(?PERSISTENT_MESSAGE_DB, ItBegin, N),
```

Comment:
Btw `{ok, end_of_stream}` possibly returned from `emqx_ds:next/3` is not handled.

Reply:
It's part of https://emqx.atlassian.net/browse/EMQX-11223